CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+# 122.0.0
+
+In this major release, we have made some API-breaking changes and landed some bug fixes and test improvements.
+
+## Breaking changes
+
+### FlexibleHeader
+
+- FlexibleHeader's topSafeAreaGuide is now a UIView.
+
+### ProgressView
+
+- We've removed generics annotations from ProgressView's progressTintColors.
+
+## API changes
```

Comment:
This can be removed if its an empty section

Reply:
I believe API diff doesn't work so well at times

Reply:
Done.